share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```
@@ -461,6 +461,28 @@ build_systems-build:
     - artifacts: True
       job: build_systems-generate
 
+###########################################
+# Build tests for different developer tools
+###########################################
+.developer_tools:
+  extends: [ ".linux_x86_64_v3" ]
+  variables:
+    SPACK_CI_STACK_NAME: developer_tools
+
+developer_tools-generate:
+  extends: [ ".developer_tools", ".generate-x86_64"]
```

```suggestion
  extends: [ ".developer-tools", ".generate-x86_64"]
```

In keeping with conventions we might want to switch to using hyphens instead of underscores here.

---

Sure, happy to make that change.